elfo-core/src/messages.rs

```diff
@@ -108,6 +108,7 @@ impl SubscribeToActorStatuses {
 }
 
 #[message]
+#[cfg_attr(feature = "test-util", derive(Constructor))]
```

Comment:
Looks like deriving constructor makes `#[non_exhaustive]` useless for creation of the type.
Writing specific constructor by hand seems more meaningful, since it won't (or shouldn't) update as definition of `ActorStatusReport` grows

Reply:
Added by hand constructor.